app/cafe.py

```diff
@@ -0,0 +1,23 @@
+
+
+import datetime
+from .errors import NotVaccinatedError, \
+    OutdatedVaccineError, NotWearingMaskError
```

Comment:
Use ( ) while importing multiple classes/modules
```suggestion
from .errors import (
    NotVaccinatedError,
    OutdatedVaccineError,
    NotWearingMaskError,
)
```